widgets/DiscordappFrame.php

```diff
@@ -1,9 +1,11 @@
 <?php
+
 namespace humhub\modules\discordapp\widgets;
 
+use humhub\models\Setting;;
 use humhub\components\Widget;
 
-class DiscordappFrame extends Widget
+class Discordapp extends \humhub\components\Widget
```

Comment:
what you doing lady? I dont understand why you rename the cass name? This will run into a error. The whole module files and code I sent was and is already perfectly working. DiscordappFrame is a class reverence and needed as is from the rest of the whole module.